pkg/rtc/participant.go

```diff
@@ -1962,7 +1962,9 @@ func (p *ParticipantImpl) onSubscriptionError(trackID livekit.TrackID) {
 }
 
 func (p *ParticipantImpl) onAnyTransportNegotiationFailed() {
-	p.params.Logger.Infow("negotiation failed, starting full reconnect")
+	if p.SinceLastSignal() < negotiationFailedTimeout {
+		p.params.Logger.Infow("negotiation failed, starting full reconnect")
+	}
```

Comment:
Instead should we check if signal connection is closed and completely skip this function? Or maybe check return of value of `IssueFullReconnect` (`writeMessage` would have to return an `error` though when `sink` is `nil`) and if that says signal channel closed, skip lopping? Just want to drive the logging based on some action.

If we want to keep this, suggesting small change to use `p.TransportManager.SinceLastSignal` to make it explicit that it is `TransportManager` API.

Reply:
In network interrupt case, the sfu normally can't detect connection closed before negotiate timeout/ice timeout if client don't send leave request, so response sink will not be nil.